src/cowboy_swagger_handler.erl

```diff
@@ -58,8 +58,13 @@ handle_get(Req, State) ->
 %%      trails routes for both: static content (Swagger-UI) and this handler
 %%      that returns the `swagger.json'.
 trails() ->
-  StaticFiles = application:get_env(
-    cowboy_swagger, static_files, "priv/swagger"),
+  CowboySwaggerPriv = cowboy_swagger_priv(),
+  SwaggerPrivDir = filename:join(CowboySwaggerPriv, "swagger"),
+  StaticFiles =
+    case application:get_env(cowboy_swagger, static_files) of
+      {ok, Val} -> Val;
+      _         -> SwaggerPrivDir
```

Comment:
The idea of evaluating `cowboy_swagger_priv()` here, instead of doing `StaticFiles = application:get_env(cowboy_swagger, static_files, cowboy_swagger_priv())` was to avoid the evaluation of said function if it wasn't actually needed. So, you should either do
```erlang
StaticFiles = application:get_env(cowboy_swagger, static_files, cowboy_swagger_priv())
```
if you don't think we need to optimize that code just yet or…
```erlang
case application:get_env(cowboy_swagger, static_files) of
  {ok, Val} -> Val;
  _         -> filename:join(cowboy_swagger_priv(), "swagger"),
end
```
…if you do ;)